prime-angular-frontend/src/app/shared/components/college-certification-form/college-certification-form.component.ts

```diff
@@ -97,6 +98,19 @@ export class CollegeCertificationFormComponent implements OnInit {
     this.remove.emit(this.index);
   }
 
+  public shouldShowPractices(): boolean {
+    if ((this.collegeCode.value == CollegeLicenceClass.BCCNM) &&
```

Comment:
Feels like this could be simplified down to a return statement since the result of the if-statement is a boolean